src/discord/commands/rooms.ts

```diff
@@ -0,0 +1,61 @@
+import { CommandInteraction, MessageActionRow, MessageEmbed, MessageSelectMenu, MessageButton, SelectMenuInteraction } from 'discord.js';
+import { Types } from 'mongoose';
+import Room from '../../models/Room';
+import Section from '../../models/section.model';
+
+async function retrieveRooms(seletedRoomId: string) {
```

Comment:
It would make more sense to type this as `selectedRoomId?: string` and adjust the rest of the code accordingly (using optional parameter rather than needing to provide an empty string)